types/models.d.ts

```diff
@@ -565,6 +569,10 @@ declare module 'mongoose' {
       TRawDocType,
       'findOneAndDelete'
     >;
+    findByIdAndRemove<ResultDoc = THydratedDocumentType>(
```

Comment:
`findByIdAndRemove()` is no longer supported in Mongoose 8, so please remove this

Reply:
Ok, should I delete all the `findByIdAndRemove` type definitions?

Reply:
I have delete all type definations for `findByIdAndRemove`